x-pack/plugins/search_playground/public/types.ts

```diff
@@ -49,6 +49,8 @@ export enum ChatFormFields {
   indices = 'indices',
   elasticsearchQuery = 'elasticsearch_query',
   summarizationModel = 'summarization_model',
+  sourceFields = 'source_fields',
+  size = 'size',
```

Comment:
nitpick: `size` seems generic. Should we use something more specific such as `docSize`?